modules/claims_api/app/controllers/claims_api/docs/metadata_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Docs
+      class MetadataController < ::ApplicationController
+        skip_before_action(:authenticate)
+
+        def index
+          render json: {
+              meta: [
```

Comment:
are you going with meta over metadata due to json api spec?

Reply:
Yes, I wanted us to keep everything we are doing based on the json API spec to make it easier to consume.